.github/CONTRIBUTING.md

```diff
@@ -1,40 +1,45 @@
 # How to contribute
 
-We are really glad you are reading this contribution guide. This means you care about the quality of your contributions.
+We are really glad that you are reading this contribution guide.
+This means you care about the quality of your contributions.
 
 ## Submitting patches via Github
 
 ### Prerequisite for submitting patches/PR's
 
-1. Basic knowledge about how to us Git(hub)
-  * Properly setting up git with `git config`
+1. Basic knowledge about how to use Git(hub)
+  * [Properly](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup) setup your git client.
   * Know how to create a commit (for details see below).
   * Know how to work with git history (rebasing your commits).
-  * Know how to create Pull Requests
+  * Know how to create Pull Requests.
 2. Have access to an Alpine Linux [development envirorment](https://wiki.alpinelinux.org/wiki/Developer_Documentation#Development).
-3. Please __do not__ submit PR's via Github's web interface. You should have a working [development envirorment](https://wiki.alpinelinux.org/wiki/Developer_Documentation#Development) available and submit your commits from the git interface.
+3. Please __do not__ submit PR's via Github's web interface. You should have a working [development envirorment](https://wiki.alpinelinux.org/wiki/Developer_Documentation#Development) available and submit your commits from your git client.
 
 ### Creating a Pull Request (PR)
 
-1. [Fork](https://help.github.com/articles/fork-a-repo/) our alpinelinux aports repository.
-2. Clone your copy of aports `git clone https://github.com/username/aports`
-3. Create a feature branch `git checkout -b my_new_feature`
-4. Make your desired changes and commit them with a correct commit message.
-  * Begin the commit message with a single short (less than 78 character) line summarizing the changes, followed by a blank line and then a more thorough description. Examples of correct formated summary lines are:
+1. [Fork](https://help.github.com/articles/fork-a-repo/) our Alpine Linux aports repository.
+2. Clone your copy of aports `git clone https://github.com/username/aports`.
+3. Create a feature branch `git checkout -b my_new_feature`.
+4. Make your desired changes and commit them with a [correct commit message](https://git-scm.com/book/ch5-2.html#Commit-Guidelines).
+  * Begin the commit message with a single short line summarizing the changes, followed by a blank line and then a more thorough description. Examples of correctly formated summary lines are:
```

Comment:
Lets not sigh each other, there is no need for that.

Im going to merge this as I cannot find any proof of any specific limit. If I look at the gitlog on github it shows variable cut offs depending on the text. I would also like to keep a limit which is not only based on github but more in general git orientated (as i already linked to the git handbook page).

Thanks for your review.